mobile/lib/shared/ui/immich_image.dart

```diff
@@ -110,8 +111,12 @@ class ImmichImage extends StatelessWidget {
         );
       },
       errorWidget: (context, url, error) {
-        debugPrint("Error getting thumbnail $url = $error");
-        CachedNetworkImage.evictFromCache(thumbnailRequestUrl);
+        if (error is HttpExceptionWithStatus &&
+            error.statusCode >= 400 &&
```

Comment:
nit: not sure if it's valid in dart, but this might be more concise:

```dart
error.statusCode / 100 == 4
```

Reply:
Http status codes are pretty common and I think referencing 400 is more readable than just a 4.

Reply:
Yeah, either way is fine. Mostly just interested to offer different perspectives.